src/tools/run-make-support/CHANGELOG.md

```diff
@@ -10,6 +10,12 @@ changes to the support library).
 This support library will probably never reach 1.0. Please bump the minor version in `Cargo.toml` if
 you make any breaking changes or other significant changes, or bump the patch version for bug fixes.
 
+## [0.2.0] - 2024-06-09
```

Comment:
If we keep bumping minor for such tiny changes, it will be large quickly xD But it's fine.

Reply:
I didn't want to invent another versioning system, so heh